routes.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express')
+const application = express()
+const port = 8000
```

Comment:
Consider using environment variables for your port number. This would make your application more flexible and secure. [important]